### PR TITLE
[ci] Enable the Style/IfInsideElse cop in Rubocop

### PR DESCRIPTION
**Pull request by Ana06**
:bowtie:

### CODE REVIEW

File: src/api/app/models/bs_request_action_maintenance_incident.rb [hunk 1]
```diff
@@ -124,20 +124,16 @@ def _merge_pkg_into_maintenance_incident(incidentProject)
                        project: linked_project, package: linked_package}
       ret = BranchPackage.new(branch_params).branch
       new_pkg = Package.get_by_project_and_name(ret[:data][:targetproject], ret[:data][:targetpackage])
-    else
-      # a new package for all targets
-      if linkinfo && linkinfo['package']
-        if Package.exists_by_project_and_name(incidentProject.name, source_package, follow_project_links: false)
-          new_pkg = Package.get_by_project_and_name(incidentProject.name, source_package, use_source: false, follow_project_links: false)
-        else
```
Comment (DavidKang) [inline src/api/app/models/bs_request_action_maintenance_incident.rb]: Why did you remove this?

Comment (Ana06) [inline src/api/app/models/bs_request_action_maintenance_incident.rb]: To put the else and the if together, but all the code is still here. I just moved it : wink:

**Comment by marcus-h** [comment]
On 2017-01-04 07:11:48 -0800, Ana María Martínez Gómez wrote:
> > @@ -124,20 +124,16 @@ def _merge_pkg_into_maintenance_incident(incidentProject)
>                         project: linked_project, package: linked_package}
>        ret = BranchPackage.new(branch_params).branch
>        new_pkg = Package.get_by_project_and_name(ret[:data][:targetproject], ret[:data][:targetpackage])
> -    else
> -      # a new package for all targets
> -      if linkinfo && linkinfo['package']
> -        if Package.exists_by_project_and_name(incidentProject.name, source_package, follow_project_links: false)
> -          new_pkg = Package.get_by_project_and_name(incidentProject.name, source_package, use_source: false, follow_project_links: false)
> -        else
> 
> To put the else and the if together, but all the code is still here. I just moved it : wink:
> 
The else block of the innermost if statement is gone (I think that's
what he meant).

**Comment by Ana06** [comment]
@marcus-h 
> The else block of the innermost if statement is gone (I think that's
> what he meant).

Oh! right :+1:

**Comment by Ana06** [comment]
[Enable the Style/IfInsideElse cop in Rubocop](https://trello.com/c/GUYjXpAR/179-enable-the-style-ifinsideelse-cop-in-rubocop)

**Comment by DavidKang** [comment]
LGTM
